plugins/aws/aws.plugin.zsh

```diff
@@ -50,32 +50,38 @@ function acp() {
 
   # First, if the profile has MFA configured, lets get the token and session duration
   local mfa_serial="$(aws configure get mfa_serial --profile $profile)"
+  local sess_duration="$(aws configure get duration_seconds --profile $profile)"
 
   if [[ -n "$mfa_serial" ]]; then
     local -a mfa_opt
-    local mfa_token sess_duration
```

Comment:
Doesn't this line have to be `local mfa_token`, as otherwise the var isn't declared? I understand `sess_duration` is declared higher up.

Reply:
Oh, yes it should be here. My bad.